vars/commonPipelineEnvironment.groovy

```diff
@@ -49,7 +53,9 @@ class commonPipelineEnvironment implements Serializable {
         githubRepo = null
 
         influxCustomData = [:]
+        influxCustomDataTags = [:]
         influxCustomDataMap = [pipeline_data: [:], step_data: [:]]
+        influxCustomDataMapTags = [pipeline_data: [:]]
```

Comment:
To avoid complexity I would propose to omit `'influxCustomDataTags'` and instead add `'jenkins_custom_data'` to `influxCustomDataMapTags `